python/lsst/pipe/tasks/mergeDetections.py

```diff
@@ -169,24 +212,57 @@ def __init__(self, butler=None, schema=None, **kwargs):
 
         @param[in] schema     the schema of the detection catalogs used as input to this one
         @param[in] butler     a butler used to read the input schema from disk, if schema is None
+        @param[in] initInputs This a PipelineTask only argument that holds all inputs passed in
```

Comment:
`This a PipelineTask only` -> `A PipelineTask-only`

Long-term (i.e. when we remove `CmdLineTask`) we'll want to make this documentation less transitional (including making it numpydoc), but this makes sense for now.

Reply:
I put in a code comment above the docstring to change at a later time